<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowThumbnailPropertyClass</class>
 <widget class="QDialog" name="ShowThumbnailPropertyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShowThumbnailForm</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="pushButtonColorNoChecked">
       <property name="text">
        <string>No check color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonColorChecked">
       <property name="text">
        <string>Checked color</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QPushButton" name="pushButtonColorF1">
         <property name="text">
          <string>F1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonColorF2">
         <property name="text">
          <string>F2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonColorF3">
         <property name="text">
          <string>F3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonColorF4">
         <property name="text">
          <string>F4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonColorF5">
         <property name="text">
          <string>F5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonColorF6">
         <property name="text">
          <string>F6</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="pushButtonColorF7">
         <property name="text">
          <string>F7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonColorF8">
         <property name="text">
          <string>F8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonColorF9">
         <property name="text">
          <string>F9</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonColorF10">
         <property name="text">
          <string>F10</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonColorF11">
         <property name="text">
          <string>F11</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonColorF12">
         <property name="text">
          <string>F12</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="cbEnableExpandDialog">
     <property name="text">
      <string>enable expand image dialog</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,0,1">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Interval to next NG</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbCheckMillSec">
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>milli sec</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Row Count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbRowCount"/>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Initial Move Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbInitialMoveMode">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>Thumbnail Mode</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Page Mode</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Column Count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbColumnCount"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="cbViewMaster">
     <property name="text">
      <string>view Master Image</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOK">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/Qtres/flash_swf_64.PNG</normaloff>:/Qtres/flash_swf_64.PNG</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/Qtres/flash_64.PNG</normaloff>:/Qtres/flash_64.PNG</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,0,1">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>NG Sign size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbNGSignSize"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>dot</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="cbModeShowPageByPiece">
     <property name="text">
      <string>Mode to show page by piece</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sbRowCount</tabstop>
  <tabstop>sbColumnCount</tabstop>
  <tabstop>sbCheckMillSec</tabstop>
  <tabstop>sbNGSignSize</tabstop>
  <tabstop>cbViewMaster</tabstop>
  <tabstop>cbEnableExpandDialog</tabstop>
  <tabstop>cbInitialMoveMode</tabstop>
  <tabstop>pbOK</tabstop>
  <tabstop>pbApply</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
